src/diamond/handler/influxdbHandler.py

```diff
@@ -170,11 +175,24 @@ def _send(self):
                 else:
                     # build metrics data
                     metrics = []
-                    for path in self.batch:
-                        metrics.append({
-                            "points": self.batch[path],
-                            "name": path,
-                            "columns": ["time", "value"]})
+                    if self.influxdb_version == "0.8":
+                        for path in self.batch:
+                            metrics.append({
+                                "points": self.batch[path],
+                                "name": path,
+                                "columns": ["time", "value"]})
+                    elif self.influxdb_version == "0.9":
+                        for path in self.batch:
+                            # HACK: convert int to float before writing to InfluxDB
```

Comment:
please link the appropriate PR/issue on the influxdb repo here so that we can track progress.